### PR TITLE
Update readme for .NET 6 + subproject description

### PR DESCRIPTION
**Pull request by cartermp**
There's more that can be done with the contributor guide and possibly issue templates, happy to do that

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,20 @@ You can support Ionide development on [Open Collective](https://opencollective.c
 
 [![Open Collective](https://opencollective.com/ionide/donate/button.png?color=blue)](https://opencollective.com/ionide)
 
+## Description
+
+Ionide-VSCode is a VSCode plugin that turns VSCode into a fully-fledged IDE for F# development.
+
+The LSP that powers language features is located here: https://github.com/fsharp/FsAutoComplete
```
Comment (baronfel) [inline README.md:19]:
```suggestion
The LSP that powers language features is [FSAutoComplete](https://github.com/fsharp/FsAutoComplete).
```

File: README.md [hunk 2]
```diff
@@ -12,13 +12,20 @@ You can support Ionide development on [Open Collective](https://opencollective.c
...
+
+The library that powers project and script loading is located here: https://github.com/ionide/proj-info
```
Comment (baronfel) [inline README.md:21]:
```suggestion
The library that powers project and script loading is [proj-info](https://github.com/ionide/proj-info)
```

**Review by baronfel — APPROVED** [review]
Couple minor http link changes

**Comment by baronfel** [comment]
Thanks man, any/all such enhancements you want, we'll take!

**Comment by cartermp** [comment]
yeet

**Pull request merged**